lib/cookies.js

```diff
@@ -10,43 +10,28 @@
  * See the License for the specific language governing permissions and limitations under the License.
  *
  */
-
-var util = require('./util');
+var Cookies = require('js-cookie');
 
 function setCookie(name, value, expiresAt) {
-  var expiresText = '';
-  if (expiresAt) {
-    expiresText = ' expires=' + util.isoToUTCString(expiresAt) + ';';
-  }
+  var cookieOptions = {
+    path: '/'
+  };
 
-  var cookieText = name + '=' + value + '; path=/;' + expiresText;
-  setCookie._setDocumentCookie(cookieText);
+  if (expiresAt && Date.parse(expiresAt)) {
+    // Expires value can be converted to a Date object
+    cookieOptions.expires = new Date(expiresAt);
+  }
```

Comment:
As we discussed, in case of parsing failures, should we not set an expiration at all, or set a default expiration? I'm more in favor of the latter, but I'll leave it to the team to decide ;)

Reply:
Based on the unlikely event that we receive a JSON string that cannot be parsed, we're going default to a session cookie.